Src/WitsmlExplorer.Frontend/components/Modals/ServerModal.tsx

```diff
@@ -133,76 +196,186 @@ const ServerModal = (props: ServerModalProps): React.ReactElement => {
     setServer({ ...server, name: e.target.value });
   };
 
-  return (
-    <ModalDialog
-      heading={`${isAddingNewServer ? "Add" : "Edit"} server`}
-      content={
-        <>
-          <ServerAndButton>
+  useEffect(() => {
+    if (serverCredentials) {
+      setUsername(serverCredentials.username);
+      setPassword(serverCredentials.password);
+    }
+  }, [serverCredentials]);
+
+  const isCompactMode = useTheme().props.MuiCheckbox.size === "small";
+  const setStandAlonePopup: ReactElement = 
+    <React.Fragment key={"standalone"}>
+      {
+        standAlone ? <div style={standAlone?Styles.noServerText:<></>}>
+          <Typography style={Styles.noServer} token={{
+            color: colors.interactive.primaryResting,
+            fontWeight: 500,
+          }} >
+            <Icons name='new_alert' size={32} />
+            No server Connected!
+          </Typography>
+          {standAlone && <Typography color="danger"  token={{fontWeight: 500,fontSize: '1rem',}} style={Styles.errorText} >{errorMessage}</Typography>}
+        </div> : ""
+      }
+      <Section style={standAlone?Styles.noServerContent:Styles.feildname} compactMode={isCompactMode}>
+        <Label label="Server URL" style={ Styles.feildname} />
+        <TextField
+          id="url"
+          className="fieldInputs"
+          size="small"
+          variant="outlined"
+          defaultValue={server?.url}
+          error={displayUrlError}
+          helperText={displayUrlError ? "Not a valid server url" : ""}
+          fullWidth
+          inputProps={{ maxLength: 256 }}
+          onChange={onChangeUrl}
+          onBlur={runUrlValidation}
+          required />
+        <Label label="Server Name" style={Styles.feildname} />
+        <TextField
+          id="name"
+          className="fieldInputs"
+          size="small"
+          variant="outlined"
+          defaultValue={server?.name}
+          error={displayNameError}
+          helperText={displayNameError ? "A server name must have 1-64 characters" : ""}
+          fullWidth
+          inputProps={{ minLength: 1, maxLength: 64 }}
+          onBlur={runServerNameValidation}
+          onChange={onChangeName}
+          required />
+        <Label label="Username" style={Styles.feildname} />
+        <TextField
```

Comment:
The advanced configuration fields, and fields for proxy and username/passwords in servermanager are not currently needed in servermanager as it is not used in configuration of witsml-explorer serverslist. But this should be discussed with PO